std/path.d

```diff
@@ -3269,7 +3269,7 @@ body
                             /* Match for:
                              *   pattern[pi0..pi-1] ~ pattern[piRemain..$]
                              */
-                            if (pattmp.ptr == null)
+                            if (() @trusted { return pattmp.ptr; }() == null)
```

Comment:
If you don't remember the details it doesn't count. `e is null` is `@safe` for any slice expression `e`.